snap/snapcraft.yaml

```diff
@@ -18,6 +18,19 @@ slots:
     name: io.flutter.demo.gallery
 
 apps:
+  daemon:
+    daemon: simple
+    restart-condition: always
```

Comment:
I'm not familiar with how daemons work.  This won't start unless a user calls start for the daemon right?

Reply:
That's what this bit is about:

https://github.com/MirServer/mir-kiosk-snap-launch/blob/aaa58086b1f7071b0a77686f30930b7463a3d256/bin/run-daemon#L5-L14

Reply:
And this will set the default value for `daemon` accordingly (only `true` by default on Ubuntu Core):

https://github.com/MirServer/mir-kiosk-snap-launch/blob/aaa58086b1f7071b0a77686f30930b7463a3d256/meta/hooks/install#L1-L10